### PR TITLE
Spelling: Language rework

### PR DESCRIPTION
**Pull request by comradekingu**
Are the log files overwritten or replaced?

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -53,10 +53,10 @@
     <string name="nav_menu_graphs">Graphs</string>
     <string name="nav_menu_settings">Settings</string>
     <string name="nav_feedback">Feedback</string>
-    <string name="nav_menu_bitcoin_donation">Donate Bitcoin</string>
+    <string name="nav_menu_bitcoin_donation">Donate bitcoin</string>
```
Comment (thuryn) [inline app/src/main/res/values/strings.xml]: Could you please update localization by weblate?
For this specific translation - Bitcon is in general name of a crypto currency. By my opinion, there should be capital letter.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -53,10 +53,10 @@
...
 
     <!-- Toast messages -->
-    <string name="connection_not_found">Check your internet connection</string>
+    <string name="connection_not_found">Check your Internet connection</string>
```
Comment (thuryn) [inline app/src/main/res/values/strings.xml]: Could you please update localization by weblate?
For this specific translation - Internet is in general name of a specific internet. By my opinion then, there should be capital letter too.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: "Bitcoin" is the organization, "bitcoin" is the currency. Though it is voluntary, that is the distinction i have found.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: I didn't understand that, should it be "network connection"?

**Comment by comradekingu** [comment]
@thuryn Could you take a look at this? I think it is ready to go.

**Comment by thuryn** [comment]
Could you, please, make this change by weblate. I believe it's easier than resolving conflicts by PR.

**Comment by comradekingu** [comment]
@thuryn Will do

**Comment by comradekingu** [comment]
@thuryn I got this far and dove into the more technical strings. Not quite done though. Closing this.

**Pull request closed without merging**